src/Perspex.Controls/Expander.cs

```diff
@@ -0,0 +1,69 @@
+﻿using Perspex.Animation;
+using Perspex.Controls.Primitives;
+
+namespace Perspex.Controls
+{
+    public enum ExpandDirection
+    {
+        Down,
+        Up,
+        Left,
+        Right
+    }
+
+    public class Expander : HeaderedContentControl
+    {
+        static Expander()
+        {
+            PseudoClass(ExpandDirectionProperty, d => d == ExpandDirection.Down, ":ExpandDirectionDown");
```

Comment:
pseudoclasses are used in some style triggers and possible they can be used in expander customization styles.
The property selectors are used in Styling Expander.Template only, just to be more visible (this can be changed).